src/test/scala/org/apache/spark/sql/delta/DeltaSuite.scala

```diff
@@ -921,4 +923,59 @@ class DeltaSuite extends QueryTest
       checkAnswer(spark.table("sc15200test"), Seq.empty)
     }
   }
+
+  test("support for setting dataChange to false") {
+    val tempDir = Utils.createTempDir()
+
+    spark.range(100)
+      .repartition(4)
```

Comment:
super nit: this first write does not really need a repartition, right? In fact, this test does not really need "repartition" as an operation anywhere for the correctness of the test.

Reply:
I agree with you on that. I was just trying to test the situation where we have to use the dataChange option.
I will remove the repartitions.